src/preact/context.js

```diff
@@ -18,7 +37,7 @@ let context;
  * - playable: whether the playback is allowed in this vDOM area. If playback
  *   is not allow, the component must immediately stop the playback.
  *
- * @return {!PreactDef.Context<AmpContextDef.ContextType>}
+ * @return {typeof context}
```

Comment:
This feels weird to me. WDYT about "duplicating" the type rather than referencing a variable?

Reply:
Agreed. Updating.